content/docs/operating/configuration.md

```diff
@@ -220,13 +220,14 @@ A `tls_config` allows configuring TLS connections.
 ```
 
 
-### `<target_group>`
+### `<static_config>`
 
-A `target_group` allows specifying a list of targets and a common label set for them.
-They are the canonical way to specify static targets in a scrape configuration.
+A `static_confic` allows specifying a list of targets and a common label set
```

Comment:
Alright. :)
Updating now, so that we can then update the docs in master for 0.20.